arrow-libs/optics/arrow-optics/src/test/kotlin/arrow/optics/std/OptionTest.kt

```diff
@@ -31,40 +23,29 @@ class OptionTest : UnitSpec() {
       aGen = Gen.option(Gen.int()),
       bGen = Gen.int(),
       funcGen = Gen.functionAToB(Gen.int()),
-      EQA = Eq.any()
     ))
 
     testLaws(PrismLaws.laws(
       prism = Prism.none(),
       aGen = Gen.option(Gen.int()),
       bGen = Gen.create { Unit },
       funcGen = Gen.functionAToB(Gen.create { Unit }),
-      EQA = Eq.any()
     ))
 
     testLaws(IsoLaws.laws(
       iso = Iso.optionToNullable<Int>().reverse(),
       aGen = Gen.int().orNull(),
       bGen = Gen.option(Gen.int()),
-      EQA = Eq.any(),
-      EQB = Eq.any(),
       funcGen = Gen.functionAToB(Gen.option(Gen.int())),
-      bMonoid = Monoid.option(Semigroup.int())
+      EQA = { a, b -> a == b },
+      EQB = { a, b -> a == b }
```

Comment:
cannot we use the defaults here?

Reply:
The default lambda `{ a, b -> a?.equals(b) == true }` was incorrect so I replaced it with `{ a, b -> a == b }` now, which is equivalent to `a?.equals(b) ?: (b === null)`.

This makes this fix redundant.